Makefile

```diff
@@ -32,7 +32,7 @@ run-end-to-end-test:
 	docker-compose run run-tests; STATUS=$$?; \
 	docker-compose down; exit $$STATUS
 
-include docker/build.mk
+#include docker/build.mk
```

Comment:
Please remove this line or uncomment it.

Reply:
Done.